src/numpy.jl

```diff
@@ -225,3 +225,23 @@ PyObject(a::Union{LinearAlgebra.Adjoint{<:Real},LinearAlgebra.Transpose}) =
     PyReverseDims(a.parent)
 
 PyObject(a::LinearAlgebra.Adjoint) = PyObject(Matrix(a)) # non-real arrays require a copy
+
+# Alternative conversion of array views mapping parent in Julia to base in Numpy
+PyObjectWithParent(a::AbstractArray) = PyObject(a)
+
+function PyObjectWithParent(pda::PermutedDimsArray{T,N,perm}) where {T,N,perm}
+    parent = PyObjectWithParent(pda.parent)
+    # numpy.transpose is similar to PermutedDimsArray and creates a view of the data
+    pycall(parent.transpose, PyObject, perm .- 1)
```

Comment:
What if `parent` is not a numpy array?  Why can't we create a numpy array directly from `pda` as discussed?

Reply:
We can create a NumPy array directly from pda. I kept this vestigial code around for comparison. I have removed it now.

Here's an expanded version of what is currently in `runtests.jl`:

```julia
julia> A = Float64[1 2; 3 4]
2×2 Array{Float64,2}:
 1.0  2.0
 3.0  4.0

julia> B = PermutedDimsArray(A, (2,1) )
2×2 PermutedDimsArray(::Array{Float64,2}, (2, 1)) with eltype Float64:
 1.0  3.0
 2.0  4.0

julia> C = PyObject(B)
PyObject array([[1., 3.],
       [2., 4.]])

julia> D = PyArray(C)
2×2 PyArray{Float64,2}:
 1.0  3.0
 2.0  4.0

julia> D == B
true

julia> A[2,1] = 7
7

julia> A
2×2 Array{Float64,2}:
 1.0  2.0
 7.0  4.0

julia> B
2×2 PermutedDimsArray(::Array{Float64,2}, (2, 1)) with eltype Float64:
 1.0  7.0
 2.0  4.0

julia> C
PyObject array([[1., 7.],
       [2., 4.]])

julia> D
2×2 PyArray{Float64,2}:
 1.0  7.0
 2.0  4.0
```